hpms/src/main/java/gov/cms/ab2d/hpms/service/HPMSAuthServiceImpl.java

```diff
@@ -61,8 +69,10 @@ private void refreshToken(long currentTimestamp) {
                 .contentType(MediaType.APPLICATION_JSON)
                 .bodyValue(retrieveAuthRequestPayload())
                 .accept(MediaType.APPLICATION_JSON)
-                .retrieve()
-                .bodyToFlux(HPMSAuthResponse.class);
+                .exchangeToFlux(r -> {
```

Comment:
I have no idea what an r is.  Please give the lambda variable a more meaningful name.